drivers/ioexpander/gpio.c

```diff
@@ -562,13 +554,11 @@ int gpio_pin_register(FAR struct gpio_dev_s *dev, int minor)
             {
               return ret;
             }
-
-          fmt = "/dev/gpint%u";
         }
         break;
     }
 
-  snprintf(devname, 16, fmt, (unsigned int)minor);
+  snprintf(devname, 16, "/dev/gpio%u", (unsigned int) minor);
```

Comment:
```suggestion
  snprintf(devname, sizeof(devname), "/dev/gpio%u", (unsigned int) minor);
```
Using `sizeof` to make it less error prone to future changes.